sdk/python/feast/infra/aws.py

```diff
@@ -262,7 +264,9 @@ def _upload_docker_image(
         _logger.info(
             f"Pulling remote image {Style.BRIGHT + Fore.GREEN}{dockerhub_image}{Style.RESET_ALL}"
         )
-        for line in docker_client.api.pull(dockerhub_image, stream=True, decode=True):
+        for line in docker_client.api.pull(
+            dockerhub_image.replace("_", "."), stream=True, decode=True
+        ):
```

Comment:
Is this a new change(aws lambda docker image uses '_' instead of '.''s or did just never work?

Reply:
This worked as of 0.16.0, but stopped working after that.